README.md

```diff
@@ -95,26 +95,37 @@ To make calls, you must create a project and segment in the IronCore Admin Conso
 
 ### Prerequisites
 
-- [Rust toolchain](https://www.rust-lang.org/tools/install) installed.
-- [cross](https://github.com/rust-embedded/cross) installed.
-- Android SDK 29. You can get the command line SDK [here](https://developer.android.com/studio) (scroll down to "Command line tools only") and then use `sdkmanager` (found in tools/bin) to install additional prerequisites:
+- Install [Rust toolchain](https://www.rust-lang.org/tools/install).
+- Install [cross](https://github.com/rust-embedded/cross).
+- Edit `$HOME/.gradle/gradle.properties` to include `sdk.dir=PATH_TO_ANDROID_SDK`.
+- Install Android SDK 29. You can get the command line SDK [here](https://developer.android.com/studio) (scroll down to "Command line tools only"). The extracted `tools` folder must follow a specific folder hierarchy. We recommend `AndroidCLI/cmdline-tools/tools`. You are then able to use `sdkmanager` (found in `tools/bin`) to install additional prerequisites:
 
   ```bash
-  ./sdkmanager --sdk_root=PATH_TO_SDK_INSTALL_LOCATION --install "build-tools;29.0.3" platform-tools "platforms;android-29"
+  ./sdkmanager "build-tools;29.0.3" "platform-tools" "platforms;android-29"
   ```
 
-- Edit `$HOME/.gradle/gradle.properties` to point the `sdk.dir` to the location of the Android SDK.
-
 ### Building
 
-From `ironoxide-java/android`, run `build.sh`. The output AAR file will be in `android/ironoxide-android/build/outputs/aar`.
+From the repository root, run `android/build.sh`. The output AAR file will be in `android/ironoxide-android/build/outputs/aar`.
```

Comment:
Just a reminder that we either need to fix the build script or change this to run it from within the `.build.sh` directory.

Reply:
I tried to add a line at the top that sets the directory to the script's location, but it may not work in all cases. It works for me, hopefully it works for you?